### DISCUSSION TITLE
Unsure what to set DB_DATA_LOCATION to

### DISCUSSION THREAD

**Discussion by SimHoZebs** [immich-app/immich#14268] [Q&A] [answered]
I've been using Immich for a while but never had `DB_DATA_LOCATION` set in my `.env` file or `docker-compose.yml`. After updating both files based on the latest documentation, I noticed this variable is now included and defaults to `./postgres`. 

However, my Immich instance as functioned without a `./postgres` directory this whole time and I assume it's defaulted elsewhere. I'm worried that restarting the container with this default might break my setup. 

What should I set `DB_DATA_LOCATION` to, considering my setup never had it before?

P.S. Unsure if this is relevant, but I'm running Immich on a rbpi, with all the images stored on an external harddisk.

**Comment by bo0tzz** [comment, marked as answer]
You should read the release notes when you upgrade. https://github.com/immich-app/immich/releases/tag/v1.102.0

**Reply by SimHoZebs** [reply]
I feel stupid - I remember reading it then and didn't opt into it. Got confused at a later upgrade anyway. Thanks.
